.github/workflows/CI.yaml

```diff
@@ -687,3 +690,17 @@ jobs:
 
       - name: "Check fuzzer with mutation test"
         run: ./scripts/check-fuzzer.sh
+
+  check-commit-message:
+    name: check commit message
+    runs-on: ubuntu-latest
+    outputs:
+      head-commit-message: ${{ steps.get_head_commit_message.outputs.headCommitMsg }}
+    steps:
+      - name: Get repo
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Print head git commit message
+        id: get_head_commit_message
+        run: echo "::set-output name=headCommitMsg::$(git show -s --format=%s)"
```

Comment:
`::set-output` is deprecated. Use `$GITHUB_OUTPUT` instead.

https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/